test/factories/submission.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :user do
+    name "Oliver Sacks"
```

Comment:
Personal favorite.

Reply:
Yeah, somehow lots of links on the HN site are from science writers!